pkg/importer/data-processor.go

```diff
@@ -49,6 +49,9 @@ const (
 	// ProcessingPhaseConvert is the phase in which the data is taken from the url provided by the source, and it is converted to the target RAW disk image format.
 	// The url can be an http end point or file system end point.
 	ProcessingPhaseConvert ProcessingPhase = "Convert"
+	// ProcessingPhaseConvertNbd is the phase in which the data is taken from the url provided by the source, and it is converted to the target using ndb to directly stream the data into the RAW disk image format.
+	// The url can be an http end point or file system end point.
+	ProcessingPhaseConvertNbd ProcessingPhase = "ConvertNbd"
```

Comment:
I am wondering if we need this phase at all. From what I am reading the Nbdkit code is only used in the http data source. And the only place that would use that phase would be the http data source. Would it not make sense to configure the qemuOperations variable in the Info function of that data source? The data source is in the same package so you have access to that variable there.

That way you can skip implementing GetNbdkit() in every data source because you need it in the data processor.